config.sample.json

```diff
@@ -1,8 +0,0 @@
-{
-    "start_date": "2020-10-01T00:00:00Z",
-    "login_customer_ids": [{"customerId": "1234567890", "loginCustomerId": "0987654321"}],
-    "oauth_client_id":"client_id",
-    "oauth_client_secret":"client_secret",
-    "refresh_token":"refresh_token",
-    "developer_token":"developer_token"
-}
```

Comment:
We may want to keep this example file to reduce the diff between the upstream and the fork repos.